lib/cucumber/formatter/interceptor.rb

```diff
@@ -31,7 +32,7 @@ def unwrap!
         end
 
         def method_missing(method, *args, &blk)
-          @pipe.send(method, *args, &blk) || super
+          @pipe.respond_to?(method) ? @pipe.send(method, *args, &blk) : super
```

Comment:
This is pretty solid. It's also the perfect companion. Often we should be using these 2 in tandem.

Reply:
I think so :) and it's definitely safer this way